structuredefinitions/Extension-England-OrganisationRole.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Any need to specify the version of XML? This is not consistent with other files.

Reply:
Hi Kazeem, Martin asked me to add <?xml version="1.0" encoding="utf-8"?>